rskj-core/src/test/java/co/rsk/peg/BridgeIT.java

```diff
@@ -1965,7 +1967,42 @@ void getRetiringFederationCreationBlockNumber() {
         TestUtils.setInternalState(bridge, "bridgeSupport", bridgeSupportMock);
         long retiringFederationCreationBlockNumber = 42L;
         when(bridgeSupportMock.getRetiringFederationCreationBlockNumber()).thenReturn(Optional.of(retiringFederationCreationBlockNumber));
-        MatcherAssert.assertThat(bridge.getRetiringFederationCreationBlockNumber(new Object[]{}), is(retiringFederationCreationBlockNumber));
+        MatcherAssert.assertThat(bridge.getRetiringFederationCreationBlockNumber(EMPTY_ARGS_FOR_BRIDGE_CALLS), is(retiringFederationCreationBlockNumber));
+    }
+
+    @Test
+    void getRetiringFederationAddress_withNoRetiringFederation_shouldReturnEmptyString() {
+        // Arrange
+        BridgeSupport bridgeSupport = mock(BridgeSupport.class);
+        Bridge aBridge = new BridgeBuilder()
+            .contractAddress(BRIDGE_ADDRESS)
+            .constants(constants)
+            .activationConfig(activationConfig)
+            .signatureCache(signatureCache)
+            .bridgeSupport(bridgeSupport)
+            .build();
+        when(bridgeSupport.getRetiringFederation()).thenReturn(Optional.empty());
+
+        // Act & Assert
+        MatcherAssert.assertThat(aBridge.getRetiringFederationAddress(EMPTY_ARGS_FOR_BRIDGE_CALLS), is(""));
+    }
+
+    @Test
+    void getRetiringFederationAddress_withRetiringFederation_shouldReturnTheCorrectRetiringFederationAddress() {
+        // Arrange
+        BridgeSupport bridgeSupport = mock(BridgeSupport.class);
```

Comment:
The same. Use `bridgeSupport` class variable. You do not need to create a new one; when doing that, you are hiding an existing class variable, Sonar will complain.